dynamodb/dynamodb.sim.w

```diff
@@ -167,6 +168,15 @@ pub class Table_sim impl dynamodb_types.ITable {
             });
           }
 
+          // Wait until we can describe the table. This is
+          // to ensure that the table is ready to be used.
+          util.waitUntil(() => {
+            client.describeTable({
```

Comment:
If the table is not ready, this will throw. I believe you need to try/catch and return false.